plugin-api/src/main/kotlin/com/jakemoore/datakache/DataKachePlugin.kt

```
@@ -29,21 +29,26 @@ class DataKachePlugin : JavaPlugin() {
         private var controller: JavaPlugin? = null
         internal lateinit var context: DataKachePluginContext
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)